.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Create a report about an existing bug
```

Comment:
You already have a PR for this?

Reply:
yes #66 .

Perhaps because it's in my master branch...